docs/getting-started/v3-migration.md

```diff
@@ -33,20 +33,21 @@ Chart.js 3.0 introduces a number of breaking changes. Chart.js 2.0 was released
 ### Options
 
 * The dataset option `tension` was renamed to `lineTension`
+* `scales.[x/y]Axes` arrays were removed. Scales are now configured directly to `options.scales` object.
```

Comment:
Maybe add a bit at the end:

Scales are now configured directly to `options.scales` object with the object key being the scale Id.

Reply:
done